src/commcare_cloud/environment/schemas/meta.py

```diff
@@ -1,6 +1,18 @@
 import jsonobject
 
 
+class GitRepository(jsonobject.JsonObject):
+    url = jsonobject.StringProperty(required=True)
+    dest = jsonobject.StringProperty(required=True)  # relative to the code_source/external directory
+    version = jsonobject.StringProperty(default="master")
+    requirements_path = jsonobject.StringProperty(default="requirements.txt")
+
+    def to_generated_variables(self):
+        vars = self.to_json()
+        vars["dest"] = "extra/" + self.dest
```

Comment:
Just curious so I can follow along. Will `extra` be a "sources root" or be a python package itself?

I.e. if we have

```
commcare-hq/
  ...
  extras/
    ...
    my_custom_repo/
      ...
      file.py
```
will we import things from file.py like `from extras.my_custom_repo.file import *` or `from my_custom_repo.file import *` or something else? Assuming these are meant to share a runtime with commcare-hq itself, will that get bootstrapped by adding the path to the django apps to `LOCAL_INSTALLED_APPS` in localsettings?